index/handlers.js

```diff
@@ -108,6 +108,8 @@ Handlers.prototype.completed = function(message) {
     return that.routeRegexp.test(route);
   }).map(function(route) {
     return that.routeRegexp.exec(route)[1];
+  }).filter(function(namespace) {
```

Comment:
Is it possible to log somewhere when indices specified in the task will not be used?  If an invalid index route is provided, and the task isn't inserted there, maybe it wouldn't be immediately obvious to the user.  Not sure if it really matters much...